shared/javascripts/tooltip.js

```diff
@@ -1,7 +1,7 @@
 /**
  * @fileOverview This script defines a tooltip to indicate that the content is
  * not a natural element of the page.
- * 
+ *
```

Comment:
All these whitespace removals on empty lines are done by my code editor automatically, as that's how most projects prefer to keep their files. If you disagree, I can amend the PR without them.